uberon_edit.obo

```diff
@@ -193951,40 +193951,84 @@ created_by: http://orcid.org/0000-0001-9897-3238
 
 [Term]
 id: UBERON:8400001
-name: hepatic acinus periportal zone
-def: "The region of the hepatic acinus closest to the portal triad. Blood in this zone has the highest oxygen concentration of the three metabolic zones in the hepatic acinus." [PMID:28126520]
-synonym: "hepatic acinus zone 1" EXACT [PMID:28126520]
+name: hepatic acinus zone 1
+def: "The region of the hepatic acinus that is closest to the arterial blood supply and has the highest oxygen concentration. Hepatocytes in this region are primarily involved in oxidative energy metabolism." [PMID:29763190]
+synonym: "liver lobule periportal zone" RELATED [PMID:28126520]
 is_a: UBERON:0000063 ! organ subunit
-relationship: adjacent_to UBERON:0001279 ! portal triad
+is_a: UBERON:8400005 ! metabolic zone of liver
 relationship: dc-creator http://orcid.org/0000-0003-2034-601X
 relationship: part_of UBERON:0001172 ! hepatic acinus
 property_value: http://purl.org/dc/elements/1.1/date 2020-09-08T14:14:21Z xsd:dateTime
 
 [Term]
 id: UBERON:8400002
-name: hepatic acinus perivenous zone
-def: "The region of the hepatic acinus closest to the central vein of the liver lobule. Blood in this zome has the lowest oxygen concentration of the three metabolic zones in the hepatic acinus." [PMID:28126520]
-synonym: "hepatic acinus centrilobular zone" EXACT [PMID:28126520]
-synonym: "hepatic acinus pericentral zone" EXACT [PMID:28126520]
-synonym: "hepatic acinus zone 3" EXACT [PMID:28126520]
+name: hepatic acinus zone 3
+def: "The region of the hepatic acinus immediately surrounding the central veins. This region has the lowest oxygen concentration in the acinus. Hepatocytes in this region are the primary location for the biotransformation of drugs." [PMID:29763190]
+synonym: "liver lobule centrilobular region" RELATED [] {RELATED="PMID:28126520"}
```

Comment:
wrong AP for xref here

Reply:
should be database_cross_reference

Reply:
Thanks for spotting! Fixed in bec1dd0ece8421de59f5acb5a08b048317bbadf7